sdk/dotnet/Pulumi.Automation/PublicAPI.Unshipped.txt

```diff
@@ -1,4 +0,0 @@
-abstract Pulumi.Automation.Workspace.GetStackOutputsAsync(string stackName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) -> System.Threading.Tasks.Task<System.Collections.Immutable.ImmutableDictionary<string, Pulumi.Automation.OutputValue>>
-override Pulumi.Automation.LocalWorkspace.GetStackOutputsAsync(string stackName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) -> System.Threading.Tasks.Task<System.Collections.Immutable.ImmutableDictionary<string, Pulumi.Automation.OutputValue>>
```

Comment:
Perhaps restore this to original state?

Reply:
I can restore this if required - it's tidying up previously shipped apis that weren't moved from unshipped to shipped.

Reply:
Gotcha!